actions/hadoop-upgrade

```diff
@@ -212,6 +212,16 @@ fi
 # set hadoop.version in unitdata
 chlp unitdata set hadoop.version ${new_hadoop_ver}
 
+if grep hadoop-lzo-${cpu_arch} resources.yaml ; then
+        juju-log "Lzo found for ${cpu_arch}, installing"
+        juju-resources fetch hadoop-lzo-${cpu_arch}
+        if [ ! $? -eq 0 ] ; then
+                action-fail "Failed to fetch hadoop-lzo-${cpu_arch} binary"
+        fi
+        lzo_path=`juju-resources resource_path hadoop-lzo-${cpu_arch}`
+        tar -zxf ${lzo_path} -C /usr/lib/hadoop/
```

Comment:
Fixed the -q, but switching hardcoded path to $HADOOP_HOME maybe we could do after people have done testing? Reason is I've already gone and used the hardcoded path, and agree, but perhaps this should be another PR (to fix this and other hardcoded paths)?